invenio_rdm_records/services/permissions.py

```diff
@@ -240,12 +240,16 @@ class RDMRecordPermissionPolicy(RecordPermissionPolicy):
     # - Records/files are updated/deleted via drafts so we don't support
     #   using below actions.
     can_update = [Disable()]
-    can_delete = [Disable()]
+    can_delete = [
+        SystemProcess()
```

Comment:
note: the service takes care of checking the preconditions of deletion workflow actions (i.e. if the record's current deletion status is wrong), and preventing the execution if they're not fulfilled by raising an exception.
i think, admins have permissions to do any actions that are not explicitly `Disable()`d